android/app/src/main/java/app/organicmaps/sdk/util/log/LogsManager.java

```diff
@@ -217,7 +218,7 @@ public synchronized void zipLogs(@NonNull OnZipCompletedListener listener)
     }
 
     Log.i(TAG, "Zipping log files in " + mLogsFolder);
-    final Runnable task = new ZipLogsTask(mLogsFolder, mLogsFolder + ".zip", listener);
+    final Runnable task = new ZipLogsTask(Objects.requireNonNull(mLogsFolder), mLogsFolder + ".zip", listener);
```

Comment:
Probably here is ok, but I suppose it has already referenced before and will be crash if null.